src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
:+1: Sounds good. Call me crazy, but purely stylistic changes can probably sit on the back burner for a little while. :)

I'm down to help with those when the time comes, it'd be nice to get a better feel for the internals of some of these contrib plugins.